### DISCUSSION TITLE
Is it possible to extract overlapping patches for training?

### DISCUSSION THREAD

**Discussion by sarthakpati** [TorchIO-project/torchio#494] [Q&A] [answered]
For training, it would be great if I could figure out a way to extract overlapping patches. Is there such a possibility?

Cheers,
Sarthak

**Comment by fepegar** [comment]
I'm not sure what you mean. Do you mean something like using a grid sampler for training?

**Reply by sarthakpati** [reply]
Yup, something similar.

**Reply by fepegar** [reply]
What would you like, precisely?

**Reply by sarthakpati** [reply]
So, this is the idea I was looking to replicate: https://arxiv.org/pdf/2006.03394.pdf

**Reply by fepegar** [reply]
I suppose you could use the grid sampler for this, then.

**Reply by sarthakpati** [reply]
Okay, thank you!

**Reply by fepegar** [reply]
You should be able to use the grid sampler like the others in `v0.18.37`.

**Reply by fepegar** [reply]
The related issue is https://github.com/fepegar/torchio/issues/513.

**Comment by fepegar** [comment, marked as answer]
You should be able to use the grid sampler like the others in `v0.18.37`, as the feature requested in #513 has been addressed.
